src/services/item/test/index.test.ts

```diff
@@ -1253,6 +1324,42 @@ describe('Item routes tests', () => {
         });
         expect(response.statusCode).toBe(StatusCodes.OK);
       });
+      it('Filter children by Folder', async () => {
+        const member = await MEMBERS_FIXTURES.saveMember(MEMBERS_FIXTURES.BOB);
+        const { item: parent } = await saveItemAndMembership({
+          member: actor,
+          creator: member,
+          permission: PermissionLevel.Read,
+        });
+        const { item: notAFolder } = await saveItemAndMembership({
+          item: getDummyItem({ name: 'child1', type: ItemType.DOCUMENT }),
+          member,
+          parentItem: parent,
+        });
+        const { item: child2 } = await saveItemAndMembership({
+          item: getDummyItem({ name: 'child2', type: ItemType.FOLDER }),
+          member,
+          parentItem: parent,
+        });
+        const children = [child2];
+
+        const response = await app.inject({
+          method: HttpMethod.GET,
+          url: `/items/${parent.id}/children?types=folder`,
```

Comment:
Use que same syntax as for the `Accessible` ? with the Query ? Why write the query manually ?